NOTICES

```diff
@@ -11191,6 +11191,60 @@ the licensed code:
 
 -----
 
+The following software may be included in this product: fsevents. A copy of the source code may be downloaded from https://github.com/fsevents/fsevents.git. This software contains the following license and notice below:
```

Comment:
Did you mean to make these changes to `NOTICES`?

Reply:
Yes. I ran `make notices`.